requirements.txt

```diff
@@ -8,7 +8,7 @@ configparser==3.5.0
 filelock==3.0.10
 flask==1.0.2
 flask-api==1.0
-hpOneView==4.7.0
+hpOneView==5.3.0
 jsonschema==2.6.0
 paste==2.0.3
 pbr==4.2.0 # Apache-2.0
```

Comment:
Can you update the validators version. Not sure why this is not showing as changes.

validators==0.12.6